README.md

```diff
@@ -180,17 +180,35 @@ The `errors` attribute contains a list of all original values that failed the un
 
 ### Error messages
 
-By default, the `ValidatorError` message will be the formatted value of `mongoose.Error.messages.general.unique` (which is set automatically by this package if not already defined).
+By default, the `ValidatorError` message will be the formatted value of a String `options.message` (which is set automatically by this package if not already defined).
 
-The default value of `mongoose.Error.messages.general.unique` is ``"Path `{PATH}` ({VALUE}) is not unique."`` which adheres to [the Mongoose error message defaults](https://github.com/Automattic/mongoose/blob/master/lib/error/messages.js).
+The default value of `options.message` is ``"Path `{PATH}` ({VALUE}) is not unique."`` which adheres to [the Mongoose error message defaults](https://github.com/Automattic/mongoose/blob/master/lib/error/messages.js).
```

Comment:
If we are going to introduce more API surface, I think we should be extra-clear on when one mechanism for overriding the message should be used instead of the other one. Currently, I have the feeling that `options.message` and the custom message in the `unique` field might feel redundant to the users.

In my opinion, the main use for `options.message` is for when the plugin is used globally. If you want to, I can try to improve the docs after merging this PR. I think renaming `options.message` to `options.defaultMessage` will make things clearer (see below).